src/storage/src/source/source_reader_pipeline.rs

```diff
@@ -266,7 +268,6 @@ fn source_reader_operator<G, C, R>(
     config: RawSourceCreationConfig,
     source_connection: C,
     connection_context: ConnectionContext,
-    reclock_follower: ReclockFollower<R::Time, mz_repr::Timestamp>,
```

Comment:
this means we can get rid of the `Rc` in `ReclockFollower` now right?

Reply:
yes, but it turns out we don't even need to store a trace at all! stay tuned for the reclock as flatmap PR